setup.cfg

```
@@ -33,7 +33,7 @@ install_requires =
     oyaml
     pyyaml >=5.4.1
     # https://github.com/audeering/audformat/pull/101
-    pandas >=1.1.5,!=1.3.0,!=1.3.1,!=1.3.2,!=1.3.3
+    pandas >=1.1.5,!=1.3.0,!=1.3.1,!=1.3.2,!=1.3.3,<1.4.0
```

As we are still hoping for a fix, should we maybe use 
```suggestion
    pandas >=1.1.5,!=1.3.0,!=1.3.1,!=1.3.2,!=1.3.3,!=1.4.0
```
instead?

---

Since we do not know if it will be fixed with `1.4.1` it might be a bit risky.

---

OK, you are maybe right. The best option would be to fix the error, but here i also agree that it is not worth the effort and we should first wait if `pandas` fixes it.

So let's stay with `<1.4.0`